src/Microsoft.Data.Sqlite/Utilities/NativeLibraryLoader.cs

```diff
@@ -81,7 +81,17 @@ public static bool IsLoaded(string dllName)
         }
 
         private static bool IsDNX()
-            => PlatformServices.Default?.Application.RuntimeFramework.Identifier == "DNX";
+        {
+            dynamic defaultPlatformServices =
```

Comment:
No good. Too many nuances. We'll stick with this hack for now. This code path may go away with CLI.